plugins/env/app/models/environment_variable_group.rb

```diff
@@ -21,7 +21,7 @@ def variable_names
     environment_variables.sort_by(&:id).map(&:name).uniq
   end
 
-  def as_json
-    super(methods: [:variable_names])
+  def as_json(options = nil)
```

Comment:
This could be `as_json(options = {})`, then we can just do `...merge(options)` below.